src/tabs/AOCTab/AnswerScripts/Day1Script.jsx

```diff
@@ -0,0 +1,3 @@
+const Day1Solution = "Day 1 Solution goes here"

```

Comment:
`Day1Solution` will most likely be a function that you call in order to get the string solution, so I would change it to be as close in functionality as possible (do this for day 2 and 3 as well)

```suggestion
const Day1Solution = () => {
  return "Day 1 Solution goes here";
};
```

Reply:
"stubbing" still requires the objects to function the same way as they will in a real environment but just sends fake data instead of real data